src/shared/components/iframe-runtime.tsx

```diff
@@ -26,12 +26,13 @@ interface IProps {
   addLocalLinkedDataListener?: (request: IAddLinkedInteractiveStateListenerRequest, phone: IframePhone) => void;
   scale?: number;
   onUnloadCallback?: (state: any) => void;
+  scrolling?: string;
```

Comment:
Perhaps:
```suggestion
  scrolling?: "auto" | "yes" | "no";
```

Reply:
@kswenson Good point.  Fixed in https://github.com/concord-consortium/question-interactives/pull/194/commits/a07903f6b5b8f8346659fc71ec167ec10043789a.